### PR TITLE
fix: update page url of obtaining access token of crowdin.

### PR DESCRIPTION
**Pull request by lyricat**
## Motivation

The page url that to obtain access token has been changed. The old url is not accessible. 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

Test it manually.

## Related PRs

No.

### CODE REVIEW

File: website/docs/i18n/i18n-crowdin.mdx [hunk 1]
```diff
@@ -163,7 +163,7 @@ We advise to:
 
 The `api_token_env` attribute defines the **env variable name** read by the Crowdin CLI.
 
-You can obtain a `Personal Access Token` on [your personal profile page](https://crowdin.com/settings#api-key).
+You can obtain a `Personal Access Token` on your personal profile page by clicking your profile photo and selecting "Account Settings - Access Tokens".
```
Comment (lex111) [inline website/docs/i18n/i18n-crowdin.mdx:166]: Weird, this is what I see if I follow the link. Everything that was there before.

image

Comment (lyricat) [reply website/docs/i18n/i18n-crowdin.mdx:166]: Ah, do you have multiple organizations? I have multiple organizations and the primary organization url is <org_name>.crowdin.com. Visiting https://crowdin.com/settings always bring me to the login page.

Comment (lyricat) [reply website/docs/i18n/i18n-crowdin.mdx:166]: I think it's cause by multiple organizations. 

BTW, according to https://support.crowdin.com/enterprise/cli/#configuration, i must specify `base_url` in crowdin.yaml to make sure all requests send to the correct API endpoints.

Comment (lex111) [reply website/docs/i18n/i18n-crowdin.mdx:166]: No, I just have personal account.

Comment (lyricat) [reply website/docs/i18n/i18n-crowdin.mdx:166]: Well. that would change a lot of things if the document need to covers many situations includes the multiple accounts. Perhaps we can close the issue and maybe sometime we will add more content in the future. How do you think?

Comment (lex111) [reply website/docs/i18n/i18n-crowdin.mdx:166]: Alright, no problem.

**Pull request closed without merging**